.env.example

```
@@ -9,3 +9,5 @@ CLIPROXY_MANAGEMENT_PASSWORD=your-management-key-here
 CLAWDBOT_TELEGRAM_TOKEN=123456789:ABCdefGHIjklMNOpqrsTUVwxyz
 # Get your Anthropic API key from: https://console.anthropic.com/
 CLAWDBOT_ANTHROPIC_KEY=sk-ant-api03-your-key-here
+# Gateway token for remote mode clients (from kyber: cat ~/.config/clawdbot/gateway-token)
+CLAWDBOT_GATEWAY_TOKEN=your-gateway-token-here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLAWDBOT_GATEWAY_TOKEN key should go before the CLAWDBOT_TELEGRAM_TOKEN key

(UnorderedKey)